CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## [0.23.0-dev.10](https://github.com/dashevo/platform/compare/v0.22.16...v0.23.0-dev.10) (2022-09-27)
+
+
+### ⚠ BREAKING CHANGES
+
+* **drive:** masternode identities sync from begging after restart (#542)
+* update Dash Core to 18.1.0-alpha.2 (#526)
+
+### Features
+
+* update Dash Core to 18.1.0-alpha.2 ([#526](https://github.com/dashevo/platform/issues/526))
+
+
+### Bug Fixes
+
+* **dapi-client:** metadata.get is not a function ([#531](https://github.com/dashevo/platform/issues/531))
```

Comment:
Maybe provide more details on this one? We already have an entry with this title in v0.23.0-dev.7

Reply:
It's autogenerated thing ))) PR and commit have the same title. Details are available with link